packages/dss-bridge/src/tests/Integration.t.sol

```diff
@@ -82,28 +101,83 @@ contract SimpleDomainGuest is DomainGuest {
     }
 
     function _isHost(address usr) internal override view returns (bool) {
-        return usr == address(usr);
+        return usr == address(host);
+    }
+
+    function revertNoSuccess(bool success, bytes memory response) internal pure {
+        if (!success) {
+            string memory message;
+            assembly {
+                let size := mload(add(response, 0x44))
```

Comment:
I know this is taken verbatim from dss-test but shouldn't the above line be
```
let size := mload(add(response, 0x24))
```
given that the structure of the response is {prefix: 4 bytes}{offset: 32 bytes}{size: 32 bytes}{reason message} ? And ditto for https://github.com/makerdao/dss-test/blob/6dd453e1d8da584ee97ae17d50423977f6e75c46/src/DSSTest.sol#L99

Reply:
Ah nevermind, I guess there is also the size of the response array itself to take into account, so the response looks like {size: 32 bytes}{prefix: 4 bytes}{offset: 32 bytes}{size: 32 bytes}{reason message}